### PR TITLE
feat: add response listener to rest api using store and hashmap

### PR DESCRIPTION
**Pull request by itsmdasifraza**
## Description
feat: add response listener to rest api using store and hashmap

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [x] 👍 yes
- [ ] 🙅 no, because I am lazy

### CODE REVIEW

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by pratikshinde590** [comment]
Please attach the issue link

File: src/packages/@app/pages/RestExplorer/RestExplorerPage.svelte [hunk 1]
```diff
@@ -132,6 +134,11 @@
       isLoginBannerActive = guestUser?.isBannerActive;
     }
   });
+
+  let restExplorerData: restExplorerData | undefined;
+  restExplorerDataStore.subscribe((webSocketMap) => {
```
Comment (Astitva877) [inline src/packages/@app/pages/RestExplorer/RestExplorerPage.svelte:139]: webSocketMap?

**Review by Astitva877 — APPROVED** [review]

**Pull request merged**